nevermined_gateway/identity/jwk_utils.py

```diff
@@ -0,0 +1,96 @@
+"""Helper functions to convert between eth keys and JWK
```

Comment:
does makes sense to move to `common-utils-py`? In that case we could re-use into the `sdk-py`

Reply:
Yes. I will do that once I'm working on the sdks. That will give me a better idea of what should be there